specification/purview/data-plane/Azure.Analytics.Purview.DataPolicies/preview/2023-08-01-preview/purviewDataPolicy.json

```diff
@@ -0,0 +1,616 @@
+            },
+            "headers": {
+              "x-ms-error-code": {
+                "type": "string",
+                "description": "The error code"
+              }
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Bind_DataPolicy": {
+            "$ref": "./examples/DataPolicy_Bind.json"
+          }
+        }
+      }
+    },
+    "/data-policies/{policy-name}:unbind": {
+      "post": {
+        "tags": [
+          "Unbind_DataPolicy"
+        ],
+        "description": "Unbinds the data policy identified by name from the datasource. Datasource is inferred from the policy. The policy goes into draft mode.",
+        "operationId": "Unbind_DataPolicy",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/policy-name"
+          },
+          {
+            "$ref": "#/parameters/if-match"
+          },
+          {
+            "$ref": "#/parameters/api-version"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/DataPolicy"
+            },
+            "headers": {
+              "ETag": {
+                "description": "An HTTP entity tag associated with the policy entity",
+                "type": "string"
+              }
+            }
+          },
+          "default": {
+            "description": "An error response received from the Policy Service",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/ErrorResponse"
+            },
+            "headers": {
+              "x-ms-error-code": {
+                "type": "string",
+                "description": "The error code"
+              }
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Unbind_DataPolicy": {
+            "$ref": "./examples/DataPolicy_UnBind.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "DataPolicyList": {
+      "required": [
+        "value"
+      ],
+      "description": "A paginated list of purview data policies",
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "array",
+          "description": "A list of purview data policies",
+          "items": {
+            "$ref": "#/definitions/DataPolicy"
+          }
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "Pagination link"
+        }
+      }
+    },
+    "DataPolicy": {
+      "required": [
+        "decisionRules",
+        "members",
+        "description"
+      ],
+      "type": "object",
+      "description": "Purview data policy",
+      "properties": {
+        "id": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The policy id"
+        },
```

Comment:
Not sure what this property is or why it is needed. It appears nowhere else in the API that I could find. Can you eliminate it?

Reply:
removed. It is not needed.